harbors/models.py

```diff
@@ -87,6 +104,11 @@ class Harbor(TranslatableModel):
         BoatType, verbose_name=_('Suitable boat types'), related_name='harbors', blank=True
     )
 
+    availability_level = models.ForeignKey(
+        AvailabilityLevel, null=True, blank=True, verbose_name=_('Availability level'),
+        related_name='harbors', on_delete=models.CASCADE
```

Comment:
Are you sure you want to delete harbors when deleting an availability level? `SET_NULL` or maybe `PROTECT` seem more viable options, of course I don't know anything about the service so I might be wrong :)

Reply:
oops 😅 copypaste let me down again

Reply:
very good catch!